@xen-orchestra/web-core/lib/components/ui/label-value/UiLabelValue.vue

```diff
@@ -0,0 +1,146 @@
+<!-- v3 -->
+<template>
+  <div class="ui-label-value typo-body-regular">
+    <div class="ui-label-value-grid-container">
+      <div class="label">
+        {{ label }}
+      </div>
+      <div class="value">
+        <div class="valueAddonWraper">
+          <slot name="value">
+            <div
+              v-if="value && !Array.isArray(value)"
+              v-tooltip
+              :class="ellipsis == true ? 'text-ellipsis' : 'text-wrap'"
+            >
+              {{ value }}
+            </div>
+            <UiTagsList v-else-if="Array.isArray(value) && value.length > 0" class="tagList">
+              <UiTag v-for="tag in value" :key="tag" accent="info" variant="secondary">
+                {{ tag }}
+              </UiTag>
+            </UiTagsList>
+          </slot>
+          <div v-if="slots.addons">
+            <slot name="addons" />
+          </div>
+        </div>
+
+        <div
+          v-if="slots.actions || (copyValue != undefined && (Array.isArray(copyValue) ? copyValue.length > 0 : true))"
+          class="actions"
+        >
+          <VtsCopyButton
+            v-if="
+              copyValue !== undefined &&
+              copyValue !== '' &&
+              !slots.actions &&
+              (Array.isArray(copyValue) ? copyValue.length > 0 : true)
+            "
+            :value="Array.isArray(copyValue) ? copyValue.join(', ') : copyValue"
+          />
+          <slot name="actions" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import VtsCopyButton from '@core/components/copy-button/VtsCopyButton.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import UiTag from '../tag/UiTag.vue'
+import UiTagsList from '../tag/UiTagsList.vue'
+
+const { value, copyValue } = defineProps<{
+  label: string
+  value?: string | string[]
+  ellipsis?: boolean
+  copyValue?: string | string[]
+}>()
+
+const slots = defineSlots<{
+  value?(): any
+  addons?(): any
+  actions?(): any
+}>()
+</script>
+
+<style lang="postcss" scoped>
+.ui-label-value {
+  container-type: inline-size;
+  container-name: card;
+
+  .ui-label-value-grid-container {
+    display: grid;
```

Comment:
```suggestion
    display: grid;
    align-items: start;
```
example : 
<img width="895" height="194" alt="image" src="https://github.com/user-attachments/assets/93e2123a-c2bb-4966-a4a9-8e785b177f0b" />